client/extensions/woocommerce/app/products/product-form-variations-card.js

```diff
@@ -11,60 +11,51 @@ import FoldableCard from 'components/foldable-card';
 import ProductVariationTypesForm from './product-variation-types-form';
 import FormToggle from 'components/forms/form-toggle';
 
-class ProductFormVariationCard extends Component {
-
-	static propTypes = {
-		product: PropTypes.shape( {
-			id: PropTypes.isRequired,
-			type: PropTypes.string.isRequired,
-			name: PropTypes.string,
-		} ),
-		editProduct: PropTypes.func.isRequired,
-		editProductAttribute: PropTypes.func.isRequired,
+const ProductFormVariationsCard = ( { product, editProduct, translate, editProductAttribute } ) => {
+	const handleToggle = () => {
+		if ( 'variable' !== product.type ) {
+			editProduct( product, { type: 'variable' } );
+		} else {
+			editProduct( product, { type: 'simple', attributes: null } );
```

Comment:
When we start to add non-variation attributes, this line will remove them. I don't think we have to solve this problem now, but I'd suggest adding a TODO comment above it to give yourself a reminder when you start implementing attributes.